source/fundamentals/connection/tls.txt

```diff
@@ -74,8 +74,11 @@ Configure Certificates
 Java applications that initiate TLS/SSL requests require access to
 cryptographic certificates that prove identity for the application
 itself as well as other applications with which the application
-interacts. To configure access to these certificates in your application,
-you should use the JVM Trust Store and the JVM Key Store.
+interacts. You can configure access to these certificates in your application with
+two mechanisms:
```

Comment:
Nit: you specified an exact amount of mechanisms -- I think we've been told to not to do this as it can be hard to maintain down the line.

```suggestion
the following mechanisms:
```

Reply:
Great catch!